src/ape/api/transactions.py

```diff
@@ -156,6 +156,9 @@ class ReceiptAPI(BaseInterfaceModel):
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__} {self.txn_hash}>"
 
+    def __getattr__(self, item):
+        return getattr(self.transaction, item, None)
```

Comment:
Really important thing about `__getattr__` is that it *must* raise `AttributeError` if the item does not exist, in order to allow the fallback effect of doing normal Python `__getattribute__` method, or else you end up with infinite loops

would suggest just changing it to this:
```suggestion
    def __getattr__(self, item: str):
        return getattr(self.transaction, item)
```

Reply:
[4f12806](https://github.com/ApeWorX/ape/pull/1015/commits/4f1280606c1c84d2372599fd4252f3dd07c5cb28)